pixelator.html

```diff
@@ -1,126 +1,172 @@
 <!DOCTYPE html>
```

Comment:
Just had a look at your code again. Both of us seem to use indentations of 2 spaces. So I could not realize where the difference is. Unfortunately! It should of course be easy to compare files for the owner.

Reply:
The difference is most likely because I have configured this kind of indentation where the head and the body are indented:

```<!DOCTYPE html>
<html lang="en">

  <head>
    <title>foo</title>
  </head>

  <body>
    <p>bar</p>
  </body>

</html>```

And the default is something like 

```
<!DOCTYPE html>
<html lang="en">

<head>
  <title>foo</title>
</head>

<body>
  <p>bar</p>
</body>

</html>
```